src/reservecounter.test.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable camelcase */
+import resCounter from './reservecounter.js';
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
reservecounter.test.js